### PR TITLE
Implemented IgniteClusterManager parameterized constructor

### PR DESCRIPTION
**Pull request by alexGalushka**
- loads Spring xml config file directly from file URL

### CODE REVIEW

File: src/main/java/io/vertx/spi/cluster/ignite/IgniteClusterManager.java [hunk 1]
```diff
@@ -107,6 +107,17 @@ public IgniteClusterManager(IgniteConfiguration cfg) {
...
   }
 
+  /**
+   * Creates cluster manager instance with given Spring xml configuration file.
+   * Use this constructor in order to configure cluster manager programmatically.
+   *
+   * @param configFile {@code URL} path to Spring xml configuration file.
+   */
+  @SuppressWarnings("unused")
+  public IgniteClusterManager(URL configFile) {
```
Comment (agura) [inline src/main/java/io/vertx/spi/cluster/ignite/IgniteClusterManager.java]: `java.net.URL` should be imported for successful compilation.

File: src/main/java/io/vertx/spi/cluster/ignite/IgniteClusterManager.java [hunk 2]
```diff
@@ -107,6 +107,17 @@ public IgniteClusterManager(IgniteConfiguration cfg) {
     this.cfg = cfg;
   }
 
+  /**
+   * Creates cluster manager instance with given Spring xml configuration file.
```
Comment (agura) [inline src/main/java/io/vertx/spi/cluster/ignite/IgniteClusterManager.java]: Please, use upper case for abbreviations. `XML` in this line and below.

File: src/main/java/io/vertx/spi/cluster/ignite/IgniteClusterManager.java [hunk 3]
```diff
@@ -280,6 +299,12 @@ private IgniteConfiguration loadConfiguration() {
     }
   }
 
+  private void setNodeToCfg(IgniteConfiguration cfg) {
```
Comment (agura) [inline src/main/java/io/vertx/spi/cluster/ignite/IgniteClusterManager.java]: It seems that `setNodeId` is better name choice.

**Comment by alexGalushka** [comment]
@agura All comments have been addressed.

**Comment by agura** [comment]
Merged into master.

Thanks for contribution.

**Pull request closed without merging**